src/components/organisms/AnimateMapOnboardFlow/AnimateMapOnboardFlow.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from "react";
+
+import { updateUserOnboarding } from "api/profile";
+
+import { VenueTemplate } from "types/venues";
+
+import { useUser } from "hooks/useUser";
+import { useVenueId } from "hooks/useVenueId";
+
+import { DeviceVideo } from "components/atoms/DeviceVideo";
+import { SvgLoop } from "components/atoms/SvgLoop";
+
+import { AnimateMapOnboardModal } from "./AnimateMapOnboardModal";
+
+// This flow works only for VenueTemplate.animatemap ATM
+const TEMPLATE = VenueTemplate.animatemap;
+
+const NO_STEP = Infinity;
+const BASE_STEP = 0;
+
+// time in ms before next SVG is displayed
+const DELAY = 4000;
+
+// images for the first modal
+const INFORMATION_SOURCES = [
+  "/animatemap-onboard/step-02.svg",
+  "/animatemap-onboard/step-03.svg",
+  "/animatemap-onboard/step-04.svg",
+];
+
+// images for the second modal
+const INTERACTION_SOURCES = [
+  "/animatemap-onboard/step-05.svg",
+  "/animatemap-onboard/step-06.svg",
+  "/animatemap-onboard/step-07.svg",
+];
+
+export const AnimateMapOnboardFlow: React.FC = () => {
```

Comment:
This component acts as a kind of script upon which the onboarding windows pop up and get replaced in order. Made only for `animatemap` template, but similar one (using the components inside) can be constructed for other templates as well.